src/containers/Application/ApplicationCreate.tsx

```diff
@@ -24,6 +25,8 @@ const ApplicationCreate: React.FC = () => {
     templateCode: type,
   })
 
+  usePageTitle(strings.PAGE_TITLE_NEW)
```

Comment:
```suggestion
  usePageTitle(strings.PAGE_TITLE_CREATE)
```
I think it's more descriptive